src/main/java/frc/robot/constants/Constants.java

```diff
@@ -62,6 +65,54 @@ public static class Shooter {
             new Rotation3d(0.0, 0.0, Math.toRadians(180.0)));
   }
 
+  public static class Vision {
+    public enum CameraType {
+      AprilTagCamera
+    }
+
+    public record CameraConfig(
+        String cameraName, Transform3d robotToCamera, CameraType cameraType) {}
+
+    public static final List<CameraConfig> configs =
+        List.of(
+            new CameraConfig(
+                "Cam0",
```

Comment:
We should rename cameras, including photon/calibrations, to match the current camera positions:
- left intake, right intake, left shooter, under shooter